addons/purchase/purchase.py

```diff
@@ -1489,20 +1474,21 @@ def _get_grouping_dicts(self, cr, uid, ids, context=None):
                                 values: procs to add
         """
         po_obj = self.pool.get('purchase.order')
+        date = time.strftime('%m/%d/%Y')
```

Comment:
this is a not our standard date format. Please use tools.DEFAULT_SERVER_DATE_FORMAT